dev-tools/scripts/smokeTestRelease.py

```diff
@@ -626,22 +626,23 @@ def verifyUnpacked(java, artifact, unpackPath, gitRevision, version, testArgs):
 
     print("    run tests w/ Java 11 and testArgs='%s'..." % testArgs)
     java.run_java11('./gradlew --no-daemon test %s' % testArgs, '%s/test.log' % unpackPath)
-    print("    compile jars w/ Java 11")
-    java.run_java11('./gradlew --no-daemon jar -Dversion.release=%s' % version, '%s/compile.log' % unpackPath)
-    print("    run interation tests w/ Java 11")
+    print("    run integration tests w/ Java 11")
     java.run_java11('./gradlew --no-daemon integrationTest -Dversion.release=%s' % version, '%s/itest.log' % unpackPath)
-    testSolrExample(unpackPath, java.java11_home, True)
+    print("    build binary release w/ Java 11")
+    java.run_java11('./gradlew --no-daemon dev -Dversion.release=%s' % version, '%s/compile.log' % unpackPath)
```

Comment:
Should we call this `assemble.log` or compile could still be ok.

Reply:
Done